app/controllers/admin/editions_controller.rb

```diff
@@ -83,6 +87,10 @@ def show
     end
   end
 
+  def show_locked
+    render :show_locked
+  end
```

Comment:
Minor but you can reduce this to `def show_locked; end`

Thinking about it you may not even need to define the method, but since this is the precedent with `new` below we should probably be consistent.

Reply:
Done 👍